eng/docker/fedora.30/Dockerfile

```diff
@@ -3,15 +3,16 @@
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #
 
-# Dockerfile that creates a container suitable to build dotnet-cli
-FROM microsoft/dotnet-buildtools-prereqs:fedora-28-c103199-20180628122443
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-30-39ec319-20190806171718
```

Comment:
@mthalman @MichaelSimons I am wondering if you can help me on this issue. I keep getting the following when trying to update the docker image. And I cannot find anywhere setlocale is used in install script

```
/usr/bin/sh setlocale: LC_ALL: cannot change locale (en_US.UTF-8) [/home/vsts/work/1/s/.nuget/packages/microsoft.dotnet.arcade.sdk/1.0.0-beta.19474.3/tools/Tools.proj]
```

Reply:
https://dev.azure.com/dnceng/public/_build/results?buildId=386114&view=logs&jobId=90615dc4-937f-5efa-0f88-9b5fc6f5bb91

Reply:
@wli3 - according to [this post](https://www.linuxquestions.org/questions/linux-newbie-8/bash-warning-setlocale-lc_all-cannot-change-locale-en-us-utf-8-a-4175628657/), the LANG is misspelled.

>Your locale should be `en_US.utf8` not `en-US.UTF-8`

Reply:
I changed the docker file to `ENV LANG=en_US.utf8` it still won't work. 
With the same error message  "cannot change locale (**en_US.UTF-8**)". I don't know what is trying to set en_US.UTF-8

I also find many similar problem like https://github.com/dotnet/coreclr/issues/22823 https://github.com/dotnet/corefx/pull/34385 but still don't know the cause other than "it is a docker authoring issue". Do you know what caused it?

Reply:
Digging a little bit I found this suggestion:  https://stackoverflow.com/a/55255020/3182557.  It is not the prettiest solution but it seems to work for the experiment I tried.

Reply:
still won't work :(